docs/micro-rdk/board/esp32.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Configure an ESP32 board"
+linkTitle: "esp32"
+weight: 20
+type: "docs"
+description: "Configure an esp32 board."
+images: ["/icons/components/board.svg"]
+tags: ["board", "components"]
+# SMEs: Gautham, Rand
+---
+
+{{% alert title="REQUIREMENTS" color="caution" %}}
+
+Follow the [setup guide](/installation/prepare/microcontrollers/) to prepare your ESP32 for running micro-RDK before configuring a `esp32` board.
+
+{{% /alert %}}
+
+To add an `esp32` board, navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com) and select **Raw JSON** mode.
+
+Copy the following JSON template and paste it into your configuration inside the `"components"` array:
+
+{{< tabs name="Configure an esp32 Board" >}}
+{{% tab name="JSON Template"%}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "attributes": {
+    "pins": [
+      <int>
+    ],
+    "analogs": [
+      {
+        "pin": "<number>",
```

Comment:
```suggestion
        "pin": "<GPIO-number>",
```
For more clarity esp. since this is different from other boards

Reply:
Ahh good suggestion but I realized this is actually an int like the pin number above! @gvaradarajan can you confirm this is not a string?